data/MeenuyD.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Meenu Yadav",
+  "bio": "Open-source enthusiast who enjoys dancing, listening to songs, reading spiritual books, and spends free time debating politics",
+  "type":"personal"
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Meenustwt",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Connect on Linkedin",
+      "url": "https://www.linkedin.com/in/meenu-yd/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on Hashnode",
+      "url": "https://hashnode.com/@meenuyD",
+      "icon": "SiHashnode"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/MeenuyD",
+      "icon": "FaGithub"
+    },
```

Also, remove this trailing comma `,` here.

```suggestion
    }
```